src/Reflection/ReflectionFunctionAbstract.php

```diff
@@ -597,4 +599,21 @@ public function removeParameter($parameterName)
             }
         }
     }
+
+    /**
+     * Fetch an array of all return statements found within this function.
```

Comment:
No, it shouldn't be returned - see test case https://github.com/Roave/BetterReflection/pull/206/commits/b79e30ddc2d76ba2bb3344ca21b164d0fc083a0e#diff-01e91f664ebf580f72f9ffa4bad97307R24